test/outdated/goldens/newer_versions.txt

```diff
@@ -252,18 +291,10 @@ local_package  ✗0.0.1   ✗0.0.1      ✗0.0.1      ✗0.0.1
 dev_dependencies:
 builder        ✗1.2.3   ✗1.3.0      ✗2.0.0      ✗2.0.0  
 
-transitive dependencies:
-transitive     ✗1.2.3   ✗1.3.0      ✗1.3.0      ✗2.0.0  
-transitive2    -        -           ✗1.0.0      ✗1.0.0  
-
-transitive dev_dependencies:
-dev_trans      ✗1.0.0   -           ✗1.0.0      ✗2.0.0  
-transitive3    -        -           ✗1.0.0      ✗1.0.0  
-
-3 upgradable dependencies are locked (in pubspec.lock) to older versions.
+2 upgradable dependencies are locked (in pubspec.lock) to older versions.
 To update these dependencies, use `pub upgrade`.
 
-3  dependencies are constrained to versions that are older than a resolvable version.
+2  dependencies are constrained to versions that are older than a resolvable version.
```

Comment:
Since we're not showing the transitive deps... I'm thinking we shouldn't count them... Which is why this is changed.